kolibri/plugins/learn/assets/src/views/commonLearnStrings.js

```diff
@@ -16,6 +16,19 @@ export const learnStrings = createTranslator('CommonLearnStrings', {
     message: 'Resume',
     context: 'Label for links that go to content that has been started and can be resumed',
   },
+  classesAndAssignmentsLabel: {
+    message: 'Classes and assignments',
```

Comment:
**Learn > Home > Classes & Lessons**

* There seem to be no ``main`` element (the extension is _guessing_)
* ``aria-label`` is misspelled and not applied to the ``nav`` element, but on the containing ``div``.

![Ubuntu18 (start)  Running  - Oracle VM VirtualBox_016](https://user-images.githubusercontent.com/1457929/186916749-3fdfacfc-884c-46ac-8f96-7e10336fbd16.png)